rmw_fastrtps_cpp/src/rmw_node.cpp

```diff
@@ -92,7 +93,7 @@ create_node(
     goto fail;
   }
 
-  node_handle = static_cast<rmw_node_t *>(malloc(sizeof(rmw_node_t)));
+  node_handle = static_cast<rmw_node_t *>(rmw_allocate(sizeof(rmw_node_t)));
```

Comment:
should this use `rmw_node_allocate` and `rmw_node_free` below ?

Reply:
Good catch